model/lifecycle.go

```diff
@@ -1276,6 +1283,7 @@ func AddNewBuilds(ctx context.Context, activated bool, v *Version, p *Project, t
 		// Extract the unique set of task names for the variant we're about to create
 		taskNames := tasks.ExecTasks.TaskNames(pair.Variant)
 		displayNames := tasks.DisplayTasks.TaskNames(pair.Variant)
+		activated := activatedVariants == nil || utility.StringSliceContains(activatedVariants, pair.Variant)
```

Comment:
Would it be a worthwhile performance improvement to use a map[string]bool for activatedVariants instead of a string slice? (Sorry I should've asked earlier)

Reply:
I don't think it'll make enough of a difference to make it messier with a map. The array shouldn't have more than 2 digits worth of elements